query-engine/connector-test-kit/src/test/scala/writes/nestedMutations/notUsingSchemaBase/NestedConnectOrCreateMutationSpec.scala

```diff
@@ -437,381 +437,6 @@ class NestedConnectOrCreateMutationSpec extends FlatSpec with Matchers with ApiS
     result.toString() should be("{\"data\":{\"findOneModelA\":{\"oneB\":{\"b_u\":\"B2\"}}}}")
   }
 
-  "A 1!:1! relation connectOrCreate" should "work and prevent relation violations" in {
```

Comment:
This looks like a dedicated test case for required 1:1 relations that we don't need anymore.

Reply:
Jep, that can go.